src/libs/actions/TaxRate.ts

```diff
@@ -0,0 +1,130 @@
+import Onyx from 'react-native-onyx';
```

Comment:
NAB, @kosmydel Should we also move openPolicyTaxesPage and enablePolicyTaxes from `policy.js` to this new file for consistency

@luacmartins Separating tax functions into a separate file seems like a good idea. We can consider doing the same thing with others new features like tag, category,...

Reply:
I can move it if needed. But here we have grouped also other similar commands like `enablePolicyTags`, so leaving would also make sense to me.

Reply:
I think we can leave the general "open this policy feature page" or "enable this feature" actions in the policy.js file